### ISSUE TITLE
Syntax error on heuristic.py 

### ISSUE THREAD

**Issue by ybouazza** [eeddaann/Bucket-brigade#3]
Hi, 

I get a syntax error on heuristic.py when i run the code, maybe it's a version issue of python ?

Thanks !

**Comment by ybouazza** [comment]
runfile('/Users/Younes-BOUAZZAOUI/Downloads/Bucket-brigade-master 2/heuristic.py', wdir='/Users/Younes-BOUAZZAOUI/Downloads/Bucket-brigade-master 2')
  File "/Users/Younes-BOUAZZAOUI/Downloads/Bucket-brigade-master 2/heuristic.py", line 55
    print "bin "+str(key) + ": " + str(d[key])
          ^
SyntaxError: invalid syntax

**Comment by eeddaann** [comment]
Yeah, I've developed it with python 2.7, i guess you are using python 3.x so to solve the issue you need to add replace print with print(), for example replace:
```python
print "bin "+str(key) + ": " + str(d[key])
```
With:
```python
print("bin "+str(key) + ": " + str(d[key]))
```

**Comment by ybouazza** [comment]
Great thanks !

**Comment by eeddaann** [comment]
After testing myself I have noticed that it wasn't enough, so I followed [this guide](https://docs.python.org/3/library/2to3.html) and fixed the issue in a1f7b30222b1d98ae42c0c87817c9bcdf1b2c110
